activity.go

```diff
@@ -0,0 +1,42 @@
+package gosnowth
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+)
+
+// RebuildActivityRequest values represent a request to rebuild activity tracking data.
+type RebuildActivityRequest struct {
+	UUID   string `json:"check_uuid"`
+	Metric string `json:"metric_name"`
+}
+
+// RebuildActivityResponse values represent a response to activity tracking rebuilds.
+type RebuildActivityResponse WriteRawResponse
```

Comment:
Rather than use a type alias for WriteRawResponse, you might just want to copy that structure into a new definition for RebuildActivityResponse.  Since the two processes are not really related, it is possible you might need to change one of those response types without wanting to change the other.  If the type is general, and you want those 4 fields to be used as a response in a lot of places, maybe you'd want to create a separate, more general type called something like IRONdbResponse and use it for both of these method results.